pkg/operator/certrotation/annotations.go

```diff
@@ -0,0 +1,37 @@
+package certrotation
+
+import (
+	"github.com/openshift/api/annotations"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func NewObjectMeta(name, namespace, jiraComponent, description string) metav1.ObjectMeta {
+	return metav1.ObjectMeta{
+		Namespace: namespace,
+		Name:      name,
+		Annotations: map[string]string{
+			annotations.OpenShiftComponent:   jiraComponent,
+			annotations.OpenShiftDescription: description,
+		},
+	}
+}
+
+// NeedsTLSMetadataUpdate mutates objectMeta setting necessary annotations if unset
+func NeedsTLSMetadataUpdate(meta *metav1.ObjectMeta, jiraComponent, description string) bool {
```

Comment:
A boolean sounding method shouldn't mutate.  Notice that `ensure` isn't a bool-hint like `is`, `are`, `should`, `must`, `needs`.